src/lib/helpers/print_helpers.rs

```diff
@@ -81,19 +82,19 @@ pub fn print_book(sdk: &SDKClient, market: &Pubkey, book: &Ladder) -> anyhow::Re
 }
 
 pub fn get_precision(mut target: u64) -> usize {
-    let mut prime_factors = BTreeMap::new();
-    let mut candidate = 2;
-    while target > 1 {
-        if target % candidate == 0 {
-            *prime_factors.entry(candidate).or_insert(0) += 1;
-            target /= candidate;
-        } else {
-            candidate += 1;
-        }
+    let mut fives = 0;
+    let mut twos = 0;
+    let initial = target;
+    while target > 0 && target % 5 == 0 {
```

Comment:
i think you don't need > 0 (unless initial target = 0) but fine ofc

Reply:
will infinte loop